### DISCUSSION TITLE
Properties in .37

### DISCUSSION THREAD

**Discussion by doctorwizz** [valinet/ExplorerPatcher#349] [General]
I just happen to notice if I right click on the taskbar in an empty area in my quick launch bar, nothing happens. I have to right click on the other side of the taskbar for properties to work. No big deal. Thought I'd let you know. This is is great work! I was glad to donate and will again!
Thanks!

**Comment by valinet** [comment]
I don't understand exactly what you mean. Like this?

![image](https://user-images.githubusercontent.com/6503598/141697275-c1c50bd8-7f2e-4602-b2d9-f16f7f26e737.png)

Or this?

![image](https://user-images.githubusercontent.com/6503598/141697296-7fa3f505-2654-4fe2-95ce-304dd2808fa2.png)

**Comment by doctorwizz** [comment]
I added a toolbar from a folder. It's on the left. If i right click below those icons and select Properties, nothing happens. 
Properties works if I right click on the right side of this windows taskbar by the explore things. 
![2021-11-11_14-31-21](https://user-images.githubusercontent.com/23109063/141697590-280700fe-b624-4cdd-a100-04a19a7b8661.png)

**Reply by valinet** [reply]
Okay, yeah, I got it, now I understand. Yeah, interesting, I will have a look.

**Reply by valinet** [reply]
I fixed it, I will push an update soon.

**Reply by valinet** [reply]
Thank you

**Reply by doctorwizz** [reply]
Also, that toolbar on left disappears after updating. All other setting are preserved. Is there anyway that toolbar could come back automatically after an update? 
Not really a problem. Very easy to add that bar back manually. Just wondering. Thank you!

**Reply by valinet** [reply]
I switched to killing `explorer.exe` instead of requesting it to restart, because it is more reliable. However, this has the side effect of not saving the settings made in Explorer (things like setting that toolbar, icons moved very recently in the systray etc). Thus, I advise, after you create the toolbar, to log off and log back in, for example; that will save the settings in Explorer and from there killing it won't affect it anymore. I think that is what you are actually experiencing. I will try to better mitigate this problem. Thanks.

**Reply by doctorwizz** [reply]
I saw this same thing happen on another pc that had an earlier version of EP like .34. After the .318 update, I restarted windows. Got to the desktop and it was downloading symbols. It all came back except for that toolbar. Had to add it back manually.

**Reply by valinet** [reply]
Yeah, idk, toolbars are not really supported since Windows 7, it's not the only problem that they have. Things like this may happen; it's fine on the computers where I test them, but yeah, idk...

**Reply by doctorwizz** [reply]
I just installed 37.2 from the EP update menu. Toolbar is still there. Just quick explorer restart. Great! Noting to fix. Love this update method!

**Reply by doctorwizz** [reply]
Toolbar (quick launch bar) disappeared when I updated to 38.2. Is that expected? Had to re-add it. Great work!

**Reply by valinet** [reply]
It takes 2 minutes to read the release notes: https://github.com/valinet/ExplorerPatcher/releases/tag/22000.318.38.2_3bbd227:

> Taskbar toolbar layouts are preserved when switching between Windows 10 and Windows 11 taskbars and in general (these will be reset when installing this update but should be subsequently remembered) (#395) (.2)

Way faster than asking a question here.

**Reply by valinet** [reply]
Before updating, you can click the notification toast to open the release notes in the browser and see what the new changes are, as well.

**Reply by doctorwizz** [reply]
Thx! I read some release notes on here. Just skimmed them though today. But I always read them.

**Reply by valinet** [reply]
That's really great to hear, as even though not many, I put effort into writing those as well. Check out the write up about how the latest features are implemented, it might be an interesting read as well: https://valinet.ro/2021/11/18/Functional-Windows-10-flyouts-on-Windows-11.html. Thanks and sorry if I was too hot tempered before.
